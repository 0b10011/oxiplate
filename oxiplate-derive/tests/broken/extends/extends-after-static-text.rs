use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline(r#"hello{% extends "multiple-blocks.html.oxip" %}"#)]
struct Static;

fn main() {
    print!("{}", Static);
}
