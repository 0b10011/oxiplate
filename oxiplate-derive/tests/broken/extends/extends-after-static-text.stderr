error: proc-macro derive panicked
 --> tests/broken/extends/extends-after-static-text.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: not yet implemented: Can't extend if already adding content

error[E0277]: `Static` doesn't implement `std::fmt::Display`
 --> tests/broken/extends/extends-after-static-text.rs:8:18
  |
8 |     print!("{}", Static);
  |             --   ^^^^^^ `Static` cannot be formatted with the default formatter
  |             |
  |             required by this formatting parameter
  |
  = help: the trait `std::fmt::Display` is not implemented for `Static`
  = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
  = note: this error originates in the macro `$crate::format_args` which comes from the expansion of the macro `print` (in Nightly builds, run with -Z macro-backtrace for more info)
